src/components/Login/Login.jsx

```diff
@@ -18,23 +22,43 @@ const Login = () => {
   return (
     <>
       <AppHeader />
-      <section id="login" className="panel">
-        <div className="row">
-          <label id="labelLogin" htmlFor="btnLogin">
-            Click the following login button to log into your pod at [
-            <a href={SOLID_IDENTITY_PROVIDER} target="_blank" rel="noreferrer">
-              {SOLID_IDENTITY_PROVIDER}
-            </a>
-            ]:{' '}
-          </label>
-          <LoginButton
-            oidcIssuer={SOLID_IDENTITY_PROVIDER}
-            redirectUrl={redirectUrl}
-            onError={console.error}
-          />
-          <p>Not logged in</p>
-        </div>
-      </section>
+      <Container component="main" maxWidth="xs">
+        <Box
+          sx={{
+            marginTop: 5,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            textAlign: 'center',
+            padding: '20px'
+          }}
+        >
+          <Paper elevation={2} sx={{ display: 'inline-block', mx: '2px', padding: '20px' }}>
+            <section id="login">
+              <div className="row">
```

Comment:
The CSS class "row" is no longer being used for PASS. className="row" can be removed completely.

Reply:
That entire div should be unneeded then?